pkg/api/mlflow/dao/repositories/metric.go

```diff
@@ -162,23 +169,27 @@ func (r MetricRepository) GetMetricHistories(
 		&database.Metric{},
 	).Where(
 		"metrics.run_uuid IN ?", runIDs,
-	).Limit(
-		func() int {
-			if maxResults != 0 {
-				return int(maxResults)
-			}
-			return 10000000
-		}(),
-	).Joins("JOIN runs on runs.run_uuid = metrics.run_uuid").
-		Order("runs.start_time DESC").
-		Order("metrics.run_uuid").
-		Order("metrics.key").
-		Order("metrics.step").
-		Order("metrics.timestamp").
-		Order("metrics.value")
+	).Joins(
+		"JOIN runs on runs.run_uuid = metrics.run_uuid",
+	).Order(
+		"runs.start_time DESC",
+	).Order(
+		"metrics.run_uuid",
+	).Order(
+		"metrics.key",
+	).Order(
+		"metrics.step",
+	).Order(
+		"metrics.timestamp",
+	).Order("metrics.value")
```

Comment:
let's make the formatting consitent here!

Reply:
yep. done.